src/devices/bus/coco/coco_ssc.cpp

```diff
@@ -34,15 +34,23 @@
 #include "coco_ssc.h"
 
 #include "cpu/tms7000/tms7000.h"
-#include "machine/netlist.h"
 #include "machine/ram.h"
-#include "netlist/devices/net_lib.h"
 #include "sound/ay8910.h"
 #include "sound/sp0256.h"
 
 #include "speaker.h"
 
-#define LOG_SSC 0
+#define LOG_INTERFACE   (1U <<  0)
```

Comment:
Oh, I just noticed this – you can’t use bit zero for one of your own categories, because it’s used for `LOG_GENERAL`/`LOG`.  You need to start at bit 1.